Extending/Content-Apps/index-v8.md

```diff
@@ -170,20 +170,11 @@ This is an example of how to register a content app with C# and perform your own
                     Alias = "appCake",
                     Name = "Cake",
                     Icon = "icon-cupcake",
-                    View = "/App_Plugins/MyContentApp/mycontentapp.html",
+                    View = "/App_Plugins/CakeContentApp/cakecontentapp.html",
                     Weight = 0
                 };
             
                 return cakeApp;
             }
         }
     }
-
-You will still need to add all of the files you added above but, because your C# code is adding the content app, the package.manifest file can be simplified like this:
```

Comment:
Hey @readingdancer ,

Any particular reason you deleted this part? 🙂

Reply:
Strange, that was not intentional.